### PR TITLE
add user defined kaldi feature type

### PR DESCRIPTION
**Pull request by ZihanLiao**
Add user-defined kaldi feature type, by default is 'kaldi-fbank'

### CODE REVIEW

**Review by pzelasko — APPROVED** [review]
Thanks, LGTM!

**Comment by ZihanLiao** [comment]
Seems like the assertion failed at https://github.com/lhotse-speech/lhotse/blob/master/test/test_kaldi_dirs.py#L213

**Comment by pzelasko** [comment]
You’ll need to update the jsonl files used in tests to have the new feature type.

**Comment by ZihanLiao** [comment]
> You’ll need to update the jsonl files used in tests to have the new feature type.

Changed the type in features.jsonl.gz from kaldi_native_io to kaldi-fbank

File: lhotse/kaldi.py [hunk 1]
```diff
@@ -80,6 +80,7 @@ def load_kaldi_data_dir(
     map_string_to_underscores: Optional[str] = None,
     use_reco2dur: bool = True,
     num_jobs: int = 1,
+    feature_type: str = 'kaldi-fbank'
```
Comment (csukuangfj) [inline lhotse/kaldi.py:83]: You also need to update
https://github.com/lhotse-speech/lhotse/blob/cb0b7d341fa348063c6925e5e2512a65c33206a4/lhotse/bin/modes/kaldi.py#L35-L41

You can add another option for `feature_type`.

Comment (csukuangfj) [reply lhotse/kaldi.py:83]: You forgot to update the subsequent python function.

You have to add it as an argument for the function that follows.

Comment (ZihanLiao) [reply lhotse/kaldi.py:83]: Sorry for my carelessness, see if any further modifications need to be done.

**Review by pzelasko — APPROVED** [review]
Thanks, LGTM! Please just fix the remaining style checks.

**Review by pzelasko — APPROVED** [review]

**Pull request merged**